src/Monitor/Monitor/MonitorCmdletBase.cs

```diff
@@ -87,16 +101,12 @@ public override void ExecuteCmdlet()
             string reasonPhrase = null;
             string message = null;
             string exName = null;
-            string supressWarningOrErrorValue = System.Environment.GetEnvironmentVariable(BreakingChangeAttributeHelper.SUPPRESS_ERROR_OR_WARNING_MESSAGE_ENV_VARIABLE_NAME);
             try
             {
-                bool supressWarningOrError;
-                Boolean.TryParse(supressWarningOrErrorValue, out supressWarningOrError);
-                if (!supressWarningOrError)
-                {
-                    WriteWarningWithTimestamp("The namespace for all the model classes will change from Microsoft.Azure.Management.Monitor.Management.Models to Microsoft.Azure.Management.Monitor.Models in future releases.");
-                    WriteWarningWithTimestamp("The namespace for output classes will be uniform for all classes in future releases to make it independent of modifications in the model classes.");
-                }
+                this.WriteIdentifiedWarning(
```

Comment:
WriteIdentifiedWarning("The namespace for all the model classes will change from Microsoft.Azure.Management.Monitor.Management.Models to Microsoft.Azure.Management.Monitor.Models in future releases.");
WriteIdentifiedWarning("The namespace for output classes will be uniform for all classes in future releases to make it independent of modifications in the model classes.");

Reply:
Done.